scripts/utils/verify-scripts.js

```diff
@@ -0,0 +1,115 @@
+module.exports = function(web3 = web3, artifacts = artifacts) {
+  const { getOrdersPaginated } = require("@gnosis.pm/dex-contracts/src/onchain_reading")
+  const Contract = require("@truffle/contract")
+
+  const { isOnlySafeOwner, fetchTokenInfoFromExchange, assertNoAllowances } = require("./trading_strategy_helpers")(
+    web3,
+    artifacts
+  )
+  const { getMasterCopy } = require("./internals")(web3, artifacts)
+  const { toErc20Units } = require("./printing_tools")
+  const { getDexagPrice } = require("./price-utils")(web3, artifacts)
+  const { checkNoProfitableOffer } = require("./price-utils")(web3, artifacts)
+  const pageSize = 50
+  const assert = require("assert")
+
+  const verifyCorrectSetup = async function(
+    brackets,
+    masterSafe,
+    allowanceExceptions,
+    globalPriceStorage = {},
+    logActivated = false
+  ) {
+    const log = logActivated ? (...a) => console.log(...a) : () => {}
+
+    const BatchExchangeArtifact = require("@gnosis.pm/dex-contracts/build/contracts/BatchExchange")
+    const networkId = await web3.eth.net.getId()
+    const BatchExchange = new web3.eth.Contract(BatchExchangeArtifact.abi, BatchExchangeArtifact.networks[networkId].address)
+
+    const auctionElementsDecoded = await getOrdersPaginated(BatchExchange, pageSize)
```

Comment:
Does `pageSize` mean that we only get the first 50 orders or is it some internal tweak parameter of the function that has no influence on the output?

Reply:
yes, it defintively has no influence on the output, it just modifies the pagesize and hence determines the number of requests indirectly.